lib/compat/wordpress-6.0/class-wp-theme-json-gutenberg.php

```diff
@@ -159,9 +171,39 @@ public function get_data() {
 			}
 		}
 
+		$flattened_theme_json = static::do_opt_out_of_settings( $flattened_theme_json );
+
 		wp_recursive_ksort( $flattened_theme_json );
 
 		return $flattened_theme_json;
 	}
 
+	protected static function do_opt_in_into_settings( &$context ) {
+		foreach ( static::TO_OPT_IN as $path ) {
+			// Use "unset prop" as a marker instead of "null" because
+			// "null" can be a valid value for some props (e.g. blockGap).
```

Comment:
Can `unset prop` be, or ever become, a valid value? I know it's unlikely for, say, color – but still. How about an object? It should also have a unique identity.